src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -9494,7 +9506,23 @@ namespace InternalHeatGains {
         // Using/Aliasing
         using namespace DataHeatBalance;
 
-        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
+        if (state.dataGlobal->CompLoadReportIsReq && !state.dataGlobal->isPulseZoneSizing) {
+            int TimeStepInDay = (state.dataGlobal->HourOfDay - 1) * state.dataGlobal->NumOfTimeStepInHour + state.dataGlobal->TimeStep;
+            for (int iZone = 1; iZone <= state.dataGlobal->NumOfZones; ++iZone) {
+                auto &znCLDayTS = state.dataOutRptTab->znCompLoads[iZone - 1].day[state.dataSize->CurOverallSimDay - 1].ts[TimeStepInDay - 1];
+                gatherCompLoadIntGain2(state, znCLDayTS, iZone);
+            }
+            if (state.dataHeatBal->doSpaceHeatBalanceSizing) {
+                for (int iSpace = 1; iSpace <= state.dataGlobal->NumOfZones; ++iSpace) {
+                    auto &spCLDayTS = state.dataOutRptTab->spCompLoads[iSpace - 1].day[state.dataSize->CurOverallSimDay - 1].ts[TimeStepInDay - 1];
+                    gatherCompLoadIntGain2(state, spCLDayTS, state.dataHeatBal->space(iSpace).zoneNum, iSpace);
+                }
+            }
```

Comment:
I recall something about if space heat balance is active, then the zone HB results are overwritten with the sum of the spaces. If that's true then why bother processing the zone's (line 9511) if those results are overwritten?  Is line 9516 iSpace = 1 to NumOfZones, or 1 to numSpaces?

    If (state.dataHeatBal->doSpaceHeatBalanceSizing) {
        gather spaces
    } else {
        gather zones
    }

Reply:
That's possible. Baby steps at this point. Just trying to find all of the places that apply so that the space results are really just for the space.